Source/WebKit/UIProcess/ProvisionalPageProxy.cpp

```diff
@@ -240,13 +256,31 @@ void ProvisionalPageProxy::didCreateMainFrame(FrameIdentifier frameID)
     ASSERT(!m_mainFrame);
 
     m_mainFrame = WebFrameProxy::create(m_page, m_process, frameID);
+    RefPtr previousMainFrame = m_page.mainFrame();
+
+    if (auto* opener = m_page.openerFrame()) {
+        if (auto* openerPage = opener->page()) {
+            auto parameters = m_page.creationParameters(m_page.process(), *m_drawingArea, API::WebsitePolicies::create());
```

Comment:
Can we get the website policies from somewhere rather than just making an empty one?